kubeshop/monokle
# feat: add glyph ref icons

This PR...

## Changes

- add glyph icons for incoming and outgoing refs

## Fixes

-

## How to test it

- select a resource from the navigator pane with incoming or outgoing links
- should see the relation icon in the editor source pane

## Screenshots

- 
<img width="916" alt="Screen Shot 2022-01-13 at 2 31 05 PM" src="https://user-images.githubusercontent.com/20525304/149330603-e5e147ea-9aaf-4edc-a51b-6e19fee85d88.png">


https://user-images.githubusercontent.com/20525304/149487961-529dc150-e84d-47a8-bdc9-3762c400f3a4.mp4


## Checklist
